Cargo.toml

```diff
@@ -30,10 +30,10 @@ curl-sys = "0.4.22"
 env_logger = "0.7.0"
 pretty_env_logger = { version = "0.4", optional = true }
 anyhow = "1.0"
-filetime = "0.2"
+filetime = "0.2.9"
 flate2 = { version = "1.0.3", default-features = false, features = ["zlib"] }
-fs2 = "0.4"
-git2 = "0.13.0"
+fs3 = "0.5"
```

Comment:
I'm a bit hesitant to bring in a whole new dependency for this, if the previous crate is unmaintained can the supporting code be copied into Cargo?

Reply:
I appreciate the desire to minimize the change here.  Really `fs3` is merely a fork of `fs2 with a few changes made:

1. Widen `#[cfg()]` guards to support illumos
2. Restructure tests so they can be run under stable as well as nightly (split out libtest-dependent bits)
3. Reconfigure CI so test suite runs again (successfully, of course)
4. Update to Rust 2018 (mainly `try!` to `?` conversions)

The intent and hope is to keep it in good working order.

Reply:
Makes sense that changes were necessary! I think though if the original author can't be reached I'd still personally prefer to inline the code necessary since we don't use all that much of the crate.